### PR TITLE
Adds empty-check to building removal

### PR DESCRIPTION
**Pull request by vincit-matu**
# Hitas Pull Request

# Description

Adds a check for the building being empty (== containing no apartments) to building removal, similarly as was previously only done in real estate. This also adds a hitasToast to signify the successful removal, for both real estates and buildings.

## Pull request checklist

Check the boxes for each DoD items that has been completed:

- **Frontend**
    - [ ] Changes have been tested

## Test plan

Try to remove a building which has apartments in it and notice the hitasToast telling you about it not being empty. Removing an empty building (or real estate) results in a success hitasToast.

## Tickets

This pull request resolves all or part of the following ticket(s): HT-239

### CODE REVIEW

File: frontend/src/common/components/OwnershipsList.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import {Button, IconAlertCircleFill, IconCrossCircle, IconPlus} from "hds-react";
+import {useImmer} from "use-immer";
+import {v4 as uuidv4} from "uuid";
+
+import {useGetOwnersQuery} from "../../app/services";
+import {IOwner, IOwnership} from "../models";
+import {dotted, formatOwner} from "../utils";
+import {FormInputField} from "./index";
+
+const OwnershipsList = ({apartment}) => {
```
Comment (ranta) [inline frontend/src/common/components/OwnershipsList.tsx:12]: What's this new component for? It's not used anywhere and seems a bit out of scope for this Ticket/PR (Not mentioned in the description)

Comment (vincit-matu) [reply frontend/src/common/components/OwnershipsList.tsx:12]: Yup, accidentally included in the PR 😅

**Review by ranta — APPROVED** [review]
🗑️
